laos_climate_change/emdat_processing.py

```diff
@@ -0,0 +1,74 @@
+import pandas as pd
+import os
+from os.path import exists
+
+
+def process_emdat(repo_path=".../laos-climate-change"):
+    if not exists(repo_path + "/data"):
+        os.makedirs(repo_path + "/data")
+
+    if not exists(repo_path + "/data/emdat.xlsx"):
+        raise NotImplementedError()
+
+    df = pd.read_excel(repo_path + "/data/emdat.xlsx", sheet_name="EM-DAT Data")
+
+    prob_cols = [  # noqa
+        "Drought",
+        "Earthquake",
+        "Extreme temperature",
+        "Flood",
+        "Storm",
+        "Wildfire",
+        "Volcanic activity",
+        "Mass Movement (Dry)",
+        "Mass Movement (Wet)",
+    ]  # noqa
+
+    intensity_cols = [
+        "Country",
+        "Start Year",
+        "Total Deaths",
+        "No. Injured",
+        "No. Affected",
+        "No. Homeless",
+        "Total Affected",
+        "Reconstruction Costs ('000 US$)",
+        "Reconstruction Costs, Adjusted ('000 US$)",
+        "Insured Damage ('000 US$)",
+        "Insured Damage, Adjusted ('000 US$)",
+        "Total Damage ('000 US$)",
+        "Total Damage, Adjusted ('000 US$)",
+        "CPI",
+    ]
+
+    df2 = df.copy()[intensity_cols]
```

Comment:
Rename `ISO` to `Country` here before you copy to save one line?

Reply:
Done!